fmralign/alignment_methods.py

```diff
@@ -135,7 +135,7 @@ def fit(self, X, Y):
     def transform(self, X):
         """Transform X using optimal transform computed during fit.
         """
-        return self.scale * X.dot(self.R)
+        return X.dot(self.R)
```

Comment:
Why do you remove the product by self.scale ?

Reply:
Because I changed R to scale * R so that it represents the complete transform when queried through algo.R. It gives a behaviour more similar to other classes.